src/components/citywide-stats/types.ts

```diff
@@ -0,0 +1,41 @@
+export interface DataPoint {
```

Comment:
At my work we tend to start interfaces with `I` to distinguish them from classes (making clear there's not extra functions and what not available). What do you think of that? For example, you'd get `IDataPoint` and `IMetricStats`